app/helpers/pulfalight_helper.rb

```diff
@@ -90,6 +90,45 @@ def hr_separator(args)
     safe_join(children, separator)
   end
 
+  # Overrides:
+  # https://github.com/projectblacklight/arclight/blob/v0.4.0/app/helpers/arclight_helper.rb#L18
+  # @param [SolrDocument]
+  def parents_to_links(document)
```

Comment:
comment:  These helpers have been removed from arclight starting at v1.0.0.beta1. Overrides like this are not good for future maintainability. But maybe that ship has already sailed. 🚢